File: tests/fixtures/objc_app/Tests/CalculatorTests.m
#import <XCTest/XCTest.h>
#import "Calculator.h"

@interface CalculatorTests : XCTestCase
@end

@implementation CalculatorTests
- (void)testAdd {
    XCTAssertEqual([[Calculator new] add:1 to:2], 3);
}
- (void)testSub {
    XCTAssertEqual([[Calculator new] sub:5 by:2], 3);
}
@end

File: tests/fixtures/swift_app/Tests/CalculatorTests.swift
import XCTest

final class CalculatorTests: XCTestCase {
    func testAdd() {
        XCTAssertEqual(Calculator().add(1, 2), 3)
    }
    func testSub() {
        XCTAssertEqual(Calculator().sub(5, 2), 3)
    }
}
